frontend/src/app/testResults/submissions/Submission.tsx

```diff
@@ -51,6 +52,9 @@ const Submission = () => {
             >
               Upload history
             </LinkWithQuery>
+            <div>
+              <h1 className="margin-top-2">Submission detail</h1>
```

Comment:
@lauraykerr what do you think of this title for this page?

Reply:
here is the screen for reference
<img width="982" alt="Screen Shot 2022-10-04 at 7 07 29 PM" src="https://user-images.githubusercontent.com/4952042/193952427-e7ff925a-2c60-411b-8c04-ec4f5ae63734.png">